calm/draft/2024-04/meta/core.json

```diff
@@ -112,6 +118,9 @@
         },
         "authentication": {
           "$ref": "#/defs/authentication"
+        },
```

Comment:
I think this should be the metadata definition rather than a ref to metadata, you have the actual definition outside the defs block at the bottom of the spec.

Reply:
My mistake